### ISSUE TITLE
add k8s v1.22.5 version

### ISSUE THREAD

**Issue by damingerdai** [AliyunContainerService/k8s-for-docker-desktop#187]

**Comment by linjinbao666** [comment]
```code
k8s.gcr.io/kube-apiserver:v1.22.5
k8s.gcr.io/kube-controller-manager:v1.22.5
k8s.gcr.io/kube-scheduler:v1.22.5
k8s.gcr.io/kube-proxy:v1.22.5
k8s.gcr.io/pause:3.2
k8s.gcr.io/etcd:3.4.13-0
k8s.gcr.io/coredns:1.7.0
```

**Comment by linjinbao666** [comment]
```code
k8s.gcr.io/kube-apiserver:v1.22.5
k8s.gcr.io/kube-controller-manager:v1.22.5
k8s.gcr.io/kube-scheduler:v1.22.5
k8s.gcr.io/kube-proxy:v1.22.5
k8s.gcr.io/pause:3.5
k8s.gcr.io/etcd:3.5.0-0
k8s.gcr.io/coredns:v1.8.4
```

**Comment by denverdino** [comment]
Fixed by  #190, thx
